.mk/vpp_agent.mk

```diff
@@ -1,16 +1,26 @@
-ARCH ?= $(shell uname -m)
-ifeq (${ARCH}, x86_64)
+ifeq (${BUILD_ARCH}, amd64)
   export VPP_AGENT=ligato/vpp-agent:v2.1.1
   export VPP_AGENT_DEV=ligato/dev-vpp-agent:v2.1.1
 endif
-ifeq (${ARCH}, aarch64)
+ifeq (${BUILD_ARCH}, arm64)
   export VPP_AGENT=ligato/vpp-agent-arm64:v2.1.1
   export VPP_AGENT_DEV=ligato/dev-vpp-agent-arm64:v2.1.1
 endif
 
 .PHONY: docker-vppagent-dataplane-dev-build
-docker-vppagent-dataplane-dev-build: docker-vppagent-dataplane-build
+docker-vppagent-dataplane-dev-build: docker-vppagent-dataplane-build install-qemu
```

Comment:
install qemu? why?

Reply:
This is a check done on Radoslav Dimitrov's PR 
https://circleci.com/gh/networkservicemesh/networkservicemesh/66971?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link
see error while building NSM docker images which are based on ligato/vpp-agent-arm64...

Digest: sha256:77f8c05335843a07e276846e54b9f994b532b3b1a80855af48281e91892851cb
Status: Downloaded newer image for ligato/vpp-agent-arm64:v2.1.1
 ---> 291f13a4df3f
Step 14/17 : COPY --from=build /go/bin/vppagent-firewall-nse /bin/vppagent-firewall-nse
 ---> 732e1a2939f9
Step 15/17 : RUN rm /opt/vpp-agent/dev/etcd.conf; echo 'Endpoint: "0.0.0.0:9112"' > /opt/vpp-agent/dev/grpc.conf; echo "disabled: true" > /opt/vpp-agent/dev/linux-plugin.conf
 ---> Running in 4d6ce52b2731

It would mean that you want  to run rm command inside ARM64 container but it would end with an error because rm is built for aarch64 architecture....

That is the reason why I added everywhere where have to be some processing inside ARM64 docker image that qemu-aarch64-static binary.